README.md

```diff
@@ -119,12 +120,14 @@ If you are compiling Armadillo by hand, ensure that LAPACK and BLAS are enabled.
 *See also the [C++ quickstart](doc/quickstart/cpp.md).*
 
 Since mlpack is a header-only library, installing just the headers for use in a
-C++ application is trivial.  From the root of the sources, configure and install
+C++ application is trivial. However, should an error occurs due to uninstalled dependencies, take a look at [Section 3.1](#31-additional-build-options) for this particular error and more.
```

Comment:
I would actually suggest putting this as a bit below the commands below, so e.g. if a user tries to do `cmake ..` and it fails, then the very next sentence tells them what to do and points them at Section 3.1.

Reply:
I might consider changing the text a little bit too:

```
If the `cmake ..` command fails due to unavailable dependencies, consider either using the
`-DDOWNLOAD_DEPENDENCIES=ON` option as detailed in
[the following subsection](#31-additional-build-options), or ensure that mlpack's dependencies
are installed, e.g. using the system package manager.  For example, on Debian and Ubuntu,
all relevant dependencies can be installed with `sudo apt-get install libarmadillo-dev libensmallen-dev libcereal-dev g++ cmake`.
```

Or, something along those lines.  Let me know what you think. :+1:

Reply:
Couldn't agree more :) I will apply your suggestion